site/docs/components/form-field/examples.mdx

```diff
@@ -0,0 +1,189 @@
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="LabelQuestion" >
+
+### Label as Question
+
+Form Field labels can be styled with a different intent by setting the `intent` prop. Use “sentence” intent for sentences and questions. See more about [displaying labels](/salt/patterns/forms#label-vs-question).
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="MultipleChildren" >
+
+### Multiple Children
+
+You can display multiple controls within a single Form Field. Wrap the controls in a Stack Layout for the correct styling.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="NecessityLabel" >
+
+### Necessity Label
+
+If a field is required, it can be marked with a "Required”, “Optional”, or “\*” indicator using the `necessity` prop.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="Readonly" >
+
+### Readonly
+
+Form Field can be set as read-only via the prop ‘readOnly’.
+
+- The prop should be applied to the field, not the control it contains, for expected results.
+
+- Set the field to read-only if you’re displaying values that are predefined and not editable, e.g., entry that’s populated and fixed by template selection.
+
+- Adheres to WCAG AA text contrast for legibility; read-only field content is always relevant to the form, but cannot be changed.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="Validation" >
+
+### Validation
+
+Form Fields can show validation states (warning, success, error) with the prop `validationStatus`.
+
+- The prop should be applied to the field,not the control it contains, for expected results.
+
+- Use the error state to alert the user of a critical issue that’s related to the field entry. This issue, which may jeopardize completion of the task, usually requires action from the user to resolve the error.
+
+- Display the warning state when you need to alert the user of a potential issue that won’t prevent the user from continuing with the task, but may cause errors if it’s not addressed.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="WithControls" >
+
+### With Controls
+
+Form Field can wrap any Salt UI control that is typically found in a form, such as Input, Checkbox and Radio Button components.
+
+- Form Field doesn’t have primary and secondary variants. Any variants apply to the field controls contained within Form Field and can be displayed via the components `variant` prop if supported.
+
+</LivePreview>
+<LivePreview componentName="form-field" exampleName="GroupedWithLabelLeft" >
+
+### Grouped With Left Aligned Label Placement
+
+Wrap multiple Form Fields in a Stack Layout to form a group.
+
+- Use the same label placement on each field.
+- Spread grouped props on the Form Fields to reduce code repetition.
+- Set the `role` on the Stack layout to "group" to enhance accessibility.
```

Comment:
curious the difference / preference, in some aria related docs, we use something like set `role="group"` instead of splitting